docs/resources/cluster.md

```diff
@@ -148,20 +147,19 @@ cluster_log_conf {
 }
 ```
 
-There are few more advanced attributes for S3 log delivery:
+There are a few more advanced attributes for S3 log delivery:
 
-* `destination` - S3 destination, e.g. `s3://my-bucket/some-prefix` You must configure the cluster with an instance profile and the instance profile must have write access to the destination. You cannot use AWS keys.  
-* `region` - (Optional) S3 region, e.g. `us-west-2`. Either region or endpoint must be set. If both are set, endpoint is used.
-* `endpoint` - (Optional) S3 endpoint, e.g. https://s3-us-west-2.amazonaws.com. Either region or endpoint needs to be set. If both are set, endpoint is used.
-* `enable_encryption` - (Optional) Enable server side encryption, false by default.
-* `encryption_type` - (Optional) The encryption type, it could be sse-s3 or sse-kms. It is used only when encryption is enabled and the default type is sse-s3.
-* `kms_key` - (Optional) KMS key used if encryption is enabled and encryption type is set to sse-kms.
-* `canned_acl` - (Optional) Set canned access control list, e.g. bucket-owner-full-control. If canned_cal is set, the cluster instance profile must have s3:PutObjectAcl permission on the destination bucket and prefix. The full list of possible canned ACL can be found [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl). By default only the object owner gets full control. If you are using cross account role for writing data, you may 
-  want to set bucket-owner-full-control to make bucket owner able to read the logs.
+* `destination` - S3 destination, e.g., `s3://my-bucket/some-prefix` You must configure the cluster with an instance profile, and the instance profile must have write access to the destination. You cannot use AWS keys.  
+* `region` - (Optional) S3 region, e.g. `us-west-2`. Either `region` or `endpoint` must be set. If both are set, the endpoint is used.
+* `endpoint` - (Optional) S3 endpoint, e.g. https://s3-us-west-2.amazonaws.com. Either `region` or `endpoint` needs to be set. If both are set, the endpoint is used.
+* `enable_encryption` - (Optional) Enable server-side encryption, false by default.
+* `encryption_type` - (Optional) The encryption type, it could be `sse-s3` or `sse-kms`. It is used only when encryption is enabled, and the default type is `sse-s3`.
+* `kms_key` - (Optional) KMS key used if encryption is enabled and encryption type is set to `sse-kms`.
+* `canned_acl` - (Optional) Set canned access control list, e.g. bucket-owner-full-control. If `canned_cal` is set, the cluster instance profile must have `s3:PutObjectAcl` permission on the destination bucket and prefix. The full list of possible canned ACL can be found [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl). By default, only the object owner gets full control. If you are using cross-account role for writing data, you may want to set bucket-owner-full-control to make bucket owners able to read the logs.
```

Comment:
"The full list of possible canned ACL" -> add s to ACL.

also wrap bucket-owner-full-control into backquotes: `bucket-owner-full-control`